### ISSUE TITLE
Take note of veteran_organizations state and state_code

### ISSUE THREAD

**Issue by jvcAdHoc** [department-of-veterans-affairs/va.gov-team#69317]
The `veteran_organizations` table has two columns related to state:
- the original `state` column that is populated with data from the [accreditation search](https://www.va.gov/ogc/apps/accreditation/) in `vso_reloader.rb`
- the new `state_code` for address data

Because the `state` column existed before we started, we should leave it alone.  That said, it may be helpful for our future selves to leave a note in the code to keep track of this situation.

~Task:~
~- [ ] determine if we want to keep the original column~

~if we only want the `state_code` column, then:~
~- [ ] create a pr to replace all organization state references with `state_code` and remove state updating in `vso_reloader.rb`~
~- [ ] create a migration only pr to remove the original `state` column~

~It is probably a good idea to reach out the lighthouse dash team to see if there are any downstream effects/consumers.~

**Comment by oddball-lindsay** [comment]
@jvcAdHoc did the `state` column exist before we started?  if so, @holdenhinkle advises leaving it alone.

**Comment by jvcAdHoc** [comment]
@oddball-lindsay yes,`state` existed before we started

**Comment by oddball-lindsay** [comment]
Closing this out, no longer needed.
